etc/docker/Dockerfile.triton-jetpack-focal

```diff
@@ -1,154 +1,147 @@
+    pkg-config \
+    \
     libc-ares-dev \
     libre2-dev \
     libssl-dev \
-    zlib1g-dev
+    zlib1g-dev \
+    \
+    libboost-all-dev \
+    \
+    curl \
+    nvidia-cuda-dev \
+    rapidjson-dev
 
-RUN mkdir -p ${HOME}/opt/src
+# Build grpc (and proto, etc.) from source, since we don't have a good version on focal
 RUN cd ${HOME}/opt/src && \
     git clone --recurse-submodules -b v1.52.0 --depth 1 --shallow-submodules https://github.com/grpc/grpc && \
     cd grpc && \
-    mkdir -p build && \
-    cd build && \
-    cmake .. -G Ninja \
+    cmake -S . -B build -G Ninja \
         -DgRPC_CARES_PROVIDER=package \
         -DgRPC_RE2_PROVIDER=package \
         -DgRPC_SSL_PROVIDER=package \
         -DgRPC_ZLIB_PROVIDER=package \
         -DgRPC_INSTALL=ON \
         -DgRPC_BUILD_TESTS=OFF \
         -DBUILD_SHARED_LIBS=ON \
-        -DCMAKE_BUILD_TYPE=RelWithDebInfo \
-        -DCMAKE_INSTALL_PREFIX=/opt/viam \
-        -DCMAKE_INSTALL_RPATH=/opt/viam/lib && \
-    ninja -j3 install
+        -DCMAKE_INSTALL_RPATH=/opt/viam/lib \
+        -DCMAKE_BUILD_TYPE=RelWithDebInfo && \
+    cmake --build build --target install -j3 -- -v && \
+    cmake --install build --prefix /opt/viam
 
 # Clone, build, and install the viam-cpp-sdk repo
-RUN apt-get -y --no-install-recommends install \
-    libboost-all-dev
-
 RUN cd ${HOME}/opt/src && \
-    git clone https://github.com/viamrobotics/viam-cpp-sdk.git && \
+    git clone https://github.com/viamrobotics/viam-cpp-sdk.git -b releases/v0.0.7 && \
     cd viam-cpp-sdk && \
     cmake -S . -B build -G Ninja -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCMAKE_PREFIX_PATH=/opt/viam && \
     PATH=/opt/viam/bin:$PATH cmake --build build --target install -j3 -- -v && \
     cmake --install build --prefix /opt/viam
 
-# Install the prerequisites for building the module against Triton Server
-RUN apt-get install -y --no-install-recommends \
-    nvidia-cuda-dev \
-    rapidjson-dev
-
-RUN apt-get install -y --no-install-recommends \
-    curl
-
 # Download and install the jetpack build of TritonServer and install it to /opt/tritonserver
-RUN curl -L https://github.com/triton-inference-server/server/releases/download/v2.35.0/tritonserver2.35.0-jetpack5.1.2.tgz | tar xfz - -C /opt
-
+RUN curl -L https://github.com/triton-inference-server/server/releases/download/v2.34.0/tritonserver2.34.0-jetpack5.1.tgz | tar xfz - -C /opt/tritonserver
 # Fixup inconsistent permissions issues in triton tarball
 RUN chmod -R oug+rX /opt/tritonserver/
 
+
+FROM build-deps AS build
+ARG DEBIAN_FRONTEND=noninteractive
+
 # Build and install the Viam Triton MLModelService from our
 # local state against the C++ SDK and Triton Server
 ADD . ${HOME}/opt/src/viam-mlmodelservice-triton
 RUN \
     cd ${HOME}/opt/src/viam-mlmodelservice-triton && \
     cmake -S . -B build -G Ninja \
-      -DVIAM_MLMODELSERVICE_TRITON_TRITONSERVER_ROOT=/opt/tritonserver \
+      -DVIAM_MLMODELSERVICE_TRITON_TRITONSERVER_ROOT=/opt/tritonserver/tritonserver \
```

Comment:
The layout of the triton server tarball changed between 2.34 and 2.35, we need to revert to older format.